docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,232 @@
+# vCenter Event Broker Appliance - Troubleshooting
+
+## Requirements
+
+You must log on to the VEBA appliance as root. You can do this from the console. If you want SSH access, execute the following command:
+
+```
+systemctl start sshd 
+```
+
+This turns on the SSH daemon but does not enable it to start on appliance boot. You should now be able to SSH into the VEBA appliance. 
+
+<BR>
+
+## Troubleshooting an initial deployment
+
+If VEBA is not working immediately after deployment, the first thing to do is check your Kubernetes pods. 
+
+```
+kubectl get pods -A
+```
+
+Here is the command output:
+
+```
+NAMESPACE        NAME                                   READY   STATUS          RESTARTS        AGE
+kube-system      coredns-584795fc57-hcvxh               1/1     Running              1          4d15h
+kube-system      coredns-584795fc57-hf72w               1/1     Running              1          4d15h
+kube-system      etcd-veba02                            1/1     Running              1          4d15h
+kube-system      kube-apiserver-veba02                  1/1     Running              1          4d15h
+kube-system      kube-controller-manager-veba02         1/1     Running              1          4d15h
+kube-system      kube-proxy-fj47p                       1/1     Running              1          4d15h
+kube-system      kube-scheduler-veba02                  1/1     Running              1          4d15h
+kube-system      weave-net-vs8ls                        2/2     Running              4          4d15h
+projectcontour   contour-5cddfc8f6-8hzd6                1/1     Running              1          4d15h
+projectcontour   contour-5cddfc8f6-jq7d8                1/1     Running              1          4d15h
+projectcontour   contour-certgen-f92l5                  0/1     Completed            0          4d15h
```

Comment:
Perhaps worth mentioning that the `Completed` state is correct, e.g. something like

> **Note:** The status `Completed` of the container `contour-certgen-...` is expected after a successful deployment of the appliance.

Reply:
Added as requested